tst/EnergyPlus/unit/ScheduleManager.unit.cc

```diff
@@ -1326,3 +1328,91 @@ TEST_F(EnergyPlusFixture, Schedule_GetCurrentScheduleValue_DST_RampUp_NoLeap)
         EXPECT_EQ(8760.0, HourOfYear);
     }
 }
+
+TEST_F(EnergyPlusFixture, ScheduleFileDSTtoggleOptionTest)
+{
+    // P. Shrestha - February 2022
+
+    // TODO: create this file with test data in it. (PS: Created)
+    // This test will fail until you do that.
+    fs::path scheduleFile = configured_source_directory() / "tst/EnergyPlus/unit/Resources/schedule_file1.csv";
+
+    std::string const idf_objects = delimited_string({
+        "Schedule:File,",
+        "  Test1,                   !- Name",
+        "  ,                        !- Schedule Type Limits Name",
+        "  " + scheduleFile.string() + ",              !- File Name",
+        "  1,                       !- Column Number",
+        "  0,                       !- Rows to Skip at Top",
+        "  8760,                    !- Number of Hours of Data",
+        "  Comma,                   !- Column Separator",         // Changed the "space" separators to "comma"
+        "  No,                      !- Interpolate to Timestep",
+        "  60,                      !- Minutes per item",
+        "  Yes,                     !- Adjust Schedule for Daylight Savings",
+        " ",
+        "Schedule:File,",
+        "  Test2,                   !- Name",
+        "  ,                        !- Schedule Type Limits Name",
+        "  " + scheduleFile.string() + ",              !- File Name",
+        "  1,                       !- Column Number",
+        "  0,                       !- Rows to Skip at Top",
+        "  8760,                    !- Number of Hours of Data",
+        "  Comma,                   !- Column Separator",
+        "  No,                      !- Interpolate to Timestep",
+        "  60,                      !- Minutes per item",
+        "  No,                     !- Adjust Schedule for Daylight Savings",
+        " ",
+        "Schedule:File,",
+        "  Test3,                   !- Name",
+        "  ,                        !- Schedule Type Limits Name",
+        "  " + scheduleFile.string() + ",              !- File Name",
+        "  1,                       !- Column Number",
+        "  0,                       !- Rows to Skip at Top",
+        "  8760,                    !- Number of Hours of Data",
+        "  Comma,                   !- Column Separator",
+        "  No,                      !- Interpolate to Timestep",
+        "  60,                      !- Minutes per item",
+        "  ;                     !- Adjust Schedule for Daylight Savings",
+        " ",
+    });
+
+    ASSERT_TRUE(process_idf(idf_objects));
+
+
+
+    // TODO: Add additional Schedule:File blocks above that test other possibilities to make sure they work right
+    // such as: 1.) Setting to "No"
+    //          2.) Leaving empty (should default to "No")
+    //          3.) Omitting the last field (should default to "No")
+
+    // (PS: TODO items above -- Done)
```

Comment:
Third one isn't done. See comment above.

Reply:
Right. Test 4 is now added in the new version.